src/components/LeftNav.js

```diff
@@ -3,57 +3,84 @@ import { I18n } from "@aws-amplify/core";
 import Image from "react-bootstrap/lib/Image";
 import DropdownButton from "react-bootstrap/lib/DropdownButton";
 import MenuItem from "react-bootstrap/lib/MenuItem";
```

Comment:
Ticket Item: Remove `MenuItem` components from `react-bootstrap` library and replace with semantic html `nav` and `nav` item components.

Reply:
👍